.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "useApiV1": false,
+            "name": "Debug operator",
```

Comment:
I don't use VSCode regularly, but was a bit confused with the friendly configuration name here, and how it renders in the Debug launch configuration dropdown:

![image](https://github.com/devfile/library/assets/593208/6e433357-7dbd-4075-ba71-f34cd6d0afdb)

```suggestion
            "name": "Debug parser",
```

Reply:
updated